README.md

```diff
@@ -4,6 +4,8 @@
 
 # gradle-revapi
 
+This is a fork of the well-known `gradle-revapi` plugin from https://github.com/palantir/gradle-revapi and has been moved under the `revapi` umbrella.
```

Comment:
Can you please add "For more details refer: https://github.com/revapi/revapi/issues/296"?